tests/primitives/test_base.py

```diff
@@ -19,6 +21,26 @@ def test_base_toloka_object_is_pickle_serializable(base_toloka_object):
     assert deserialized == base_toloka_object
 
 
+def test_generic_base_inherited_object():
```

Comment:
Can you please add more tests? In particular:
* Generic with 2+ typevars
* Generic with nested generic
* Test with nongeneric subclass of generic:
```
class A(Generic[T]):
    a: T
class B(A[int]):
    b: str
```